www/templates/menu.html

```diff
@@ -34,6 +34,10 @@ <h1 class="title">PVTA</h1>
           <i class="icon ion-ios-location" style="margin-right: 10px"></i>
           Stops
         </ion-item>
+        <ion-item menu-close href="#/app/map">
+          <i class="icon ion-android-map" style="margin-right: 10px"></i>
```

Comment:
This is more to set up for my next feature, which is the map that shows all buses within x miles of you